cmd/go-swagger/restapi/configure_swagger_bench.go

```diff
@@ -18,12 +18,12 @@ import (
 	"github.com/Contrast-Security-OSS/go-test-bench/cmd/go-swagger/restapi/operations/sql_injection"
 	"github.com/Contrast-Security-OSS/go-test-bench/cmd/go-swagger/restapi/operations/ssrf"
 	"github.com/Contrast-Security-OSS/go-test-bench/cmd/go-swagger/restapi/operations/swagger_server"
-	"github.com/Contrast-Security-OSS/go-test-bench/cmd/go-swagger/restapi/operations/unvalidated_redirect"
 	"github.com/Contrast-Security-OSS/go-test-bench/cmd/go-swagger/restapi/operations/xss"
 	"github.com/Contrast-Security-OSS/go-test-bench/internal/common"
 )
 
 //go:generate go run ../regen/regen.go
+//go:generate rm -rf ./operations/
```

Comment:
The order of these two needs swapped so that the files are deleted and _then_ re-created.

Reply:
I see some updates but it doesn't look like these were swapped as Mark asked for.